scripts/bridgeERC20.ts

```diff
@@ -110,18 +110,23 @@ const FUEL_FUNGIBLE_TOKEN_ADDRESS = process.env.FUEL_FUNGIBLE_TOKEN_ADDRESS || '
   console.log('');
 
   // verify compatability between the two token contracts
-  // TODO: also verify the gateway contract matches on the fuel side fungible token contract
-  const l1Decimals = parseInt('' + (await fuelTestToken.functions.layer1_decimals().dryRun()).value);
-  const actualL1Decimals = parseInt('' + (await ethTestToken.decimals()));
-  if (l1Decimals != actualL1Decimals) {
+  const l1Decimals = parseInt('' + (await fuelTestToken.functions.bridged_token_decimals().dryRun()).value);
+  const expectedL1Decimals = parseInt('' + (await ethTestToken.decimals()));
+  if (l1Decimals != expectedL1Decimals) {
     throw new Error(
-      `Expected L1 decimals from the Fuel token contract does not match the actual L1 decimals [expected:${l1Decimals}, actual:${actualL1Decimals}].`
+      `L1 decimals from the Fuel token contract does not match the actual L1 decimals [expected:${expectedL1Decimals}, actual:${l1Decimals}].`
```

Comment:
Should we update the "L1" references as per the changes made to the Bridge Fungible Token where we're using "BRIDGED_TOKEN_DECIMALS " now?

Reply:
I decided that these integration tests should use the terminology of the specific deployment being targeted. In this case it's a rollup with a clear L1 and L2. No need to be cryptic at this point